core/src/main/java/com/sequenceiq/distrox/v1/distrox/controller/DistroXV1Controller.java

```diff
@@ -90,14 +99,16 @@ public StackViewV4Responses list(String environmentName, String environmentCrn)
     }
 
     @Override
-    public StackV4Response post(@Valid DistroXV1Request request) {
+    @CheckPermissionByAccount(action = AuthorizationResourceAction.WRITE)
```

Comment:
I made the annotation as method annotation because I think we will have controller which has method both with resource level permission check and account level permission check so we need it as a method annotation, we can make it for a class too, but it requires addition logic, I decided to not implement that for now

Reply:
added to TODO list